recipes/geogram/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,278 @@
+// Copied from https://github.com/BrunoLevy/geogram/blob/main/src/tests/test_RVC/main.cpp
```

Comment:
Please, simplify it to a small test, link consuming a method that print the library version. The intention is testing the package, something small and functional. Not running examples neither unit tests that are more complex to maintain.